### ISSUE TITLE
Love it

### ISSUE THREAD

**Issue by dapinitial** [simurai/simurai.github.io#8]
Which versions of Atom does this support?

**Comment by simurai** [comment]
By "this" you probably meant these [DuoTone themes](http://simurai.com/projects/2016/01/01/duotone-themes)?

They should work with the current Atom version `1.12`. And probably also a few versions back.

**Comment by dapinitial** [comment]
Yes, I did mean the DuoThemes. I don't know what I did, I was just using
the comment button on your website. I saw that it was starting a new issue.
I thought that's how you wanted it?

**Comment by simurai** [comment]
Yeah, using GitHub issues as comments is kinda experimental. 😬 

It probably should pre-populate the title, like `[Feedback] Title of the blog post`. Will try to add that.. one day ™️.

**Comment by dapinitial** [comment]
I thought it was novel considering you're employed by github and leveraging
the technology and resources that you already are a part of! Kudos
